packages/superset-ui-legacy-plugin-chart-table/src/ReactDataTable.tsx

```diff
@@ -177,22 +151,28 @@ export default function ReactDataTable(props: DataTableProps) {
    * and adjust the pagination size (which is not configurable via DataTables API).
    */
   function drawCallback(this: DataTables.JQueryDataTables) {
-    const table = this.api();
-    const metricCols = table.columns('.dt-metric', { page: 'current' });
-    const cellData = metricCols.data();
-    metricCols.every(function(colIdx, _, colIterIdx) {
-      const key = columnOptions[colIdx].data;
-      this.nodes().each((item, i) => {
-        const val = cellData[colIterIdx][i];
-        $(item)
-          .attr('title', val)
-          .css('background-image', cellBar(key, val));
-      });
-    });
-    // force smaller pagination
+    // force smaller pagination, because datatables-bs hard-corded pagination styles
     $('.pagination', rootElem.current as HTMLElement).addClass('pagination-sm');
   }
 
+  /**
+   * Formatted text for cell value
+   */
+  function cellText(key: string, format: string | undefined, val: unknown) {
+    if (key === '__timestamp') {
+      return formatTimestamp(val);
+    } else if (typeof val === 'string') {
+      // It's fine to return raw value react will handles HTML escaping
+      return val;
```

Comment:
Could we also test if the cells can render html links correctly?

Reply:
Updated with [js-xss](https://github.com/leizongmin/js-xss) library.